src/DatabaseAdapter.js

```diff
@@ -49,6 +49,18 @@ function clearDatabaseSettings() {
   appDatabaseOptions = {};
 }
 
+//Used by tests
+function destroyAllDataPermanently() {
+  if (process.env.TESTING) {
```

Comment:
I wrapped this a little differently, can change if there is any advantage to the previously suggested way of doing this?